### DISCUSSION TITLE
3D View only for active Tool

### DISCUSSION THREAD

**Discussion by mintracer** [terjeio/ioSender#245] [Ideas]
Hey...
First of all, thank you for this really nice Software and your regardless time and work!

i'm starting to use Gcode with some toolchanges... thats fine... 

but in the 3d View all toolpath are layered. 

If the toolpath is heavy detailed the view is black... 
![IMG_20220826_232852](https://user-images.githubusercontent.com/97637453/186993274-4e6c4eaa-dfab-4f4b-9a9d-7362fbf12ac4.jpg)

so maybe its possible just to show the active Tool toolpath... 

thanks for watching ;)

cheers

Einar

**Comment by terjeio** [comment]
Added to [roadmap](https://github.com/terjeio/ioSender/wiki/Roadmap).
